packages/web-components/src/components/va-search/va-search.tsx

```diff
@@ -0,0 +1,43 @@
+import { Component, Host, h, Prop, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'va-search',
+  styleUrl: 'va-search.css',
+  shadow: true,
+})
+export class VaSearch {
+  @Prop() action: string;
+
+  @Prop() label: string = 'Search';
+
+  @Prop() method?: 'GET' | 'POST' | 'dialog';
+
+  @Event({ bubbles: true, composed: true })
+  submitEvent: EventEmitter;
+
+  private handleSubmitEvent(event: KeyboardEvent | MouseEvent) {
+    this.submitEvent.emit(event);
+  }
+
+  render() {
+    const { action, label, method } = this;
+
+    return (
+      <Host>
+        <form
+          id="va-search-form"
+          action={action}
+          method={method}
+          onSubmit={(event: KeyboardEvent | MouseEvent) =>
+            this.handleSubmitEvent(event)
+          }
+        >
+          <input id="va-search-input" aria-label={label} type="text"></input>
```

Comment:
Should we use the `va-text-input` component

Reply:
I'm against using `va-text-input` here because the native HTML element does everything we need for this search component. `va-text-input` also uses `aria-describedby` (I think we would have to create a visually hidden element with text to work around this), and we wouldn't use the error or label. 

We could make `va-text-input` more flexible and revisit this later.